src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_VALUE = 20;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final String DEFAULT_COLOR = "White";
```

Comment:
use Color.WHITE.name()

Reply:
I didn't understand this comment fully, but I did as I understood